core/src/main/java/io/substrait/expression/WindowFunctionInvocation.java

```diff
@@ -1,33 +0,0 @@
-package io.substrait.expression;
-
-import io.substrait.extension.SimpleExtension;
-import io.substrait.type.Type;
-import java.util.List;
-import java.util.Map;
-import org.immutables.value.Value;
-
-@Value.Immutable
-public abstract class WindowFunctionInvocation {
```

Comment:
WindowFunctionInvocation is a / implements `Expression`, much like ScalarFunctionInvocation.

Moved this to an inner class of Expression for consistency.